<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditAccount</class>
 <widget class="QWidget" name="EditAccount">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit account</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="accountTab">
      <attribute name="title">
       <string>Account</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>4</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="connectionTab">
      <attribute name="title">
       <string>Connection</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="QCheckBox" name="useProfileCheckBox">
         <property name="text">
          <string>Use profile settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
